orchagent/muxorch.cpp

```diff
@@ -1495,6 +1498,10 @@ bool MuxCableOrch::addOperation(const Request& request)
     auto state = request.getAttrString("state");
     auto mux_obj = mux_orch->getMuxCable(port_name);
 
+    /* Disable Pfc Watchdog for perfomance */
+    SWSS_LOG_INFO("Disabling PFCWD for Mux Switchover on %s", port_name.c_str());
+    gPfcWdSwOrch->stopWdOnAllPorts();
```

Comment:
I'm a little confused here. The mux cable state is per port, why we disable watchdog on all ports?